orchestrator/gorc/Cargo.toml

```diff
@@ -6,6 +6,8 @@ edition = "2021"
 rust-version = "1.56"
 
 [dependencies]
+aws-config = { git = "https://github.com/awslabs/aws-sdk-rust", tag = "v0.0.17-alpha", package = "aws-config" }
+aws-sdk-secretsmanager = { git = "https://github.com/awslabs/aws-sdk-rust", tag = "v0.0.17-alpha", package = "aws-sdk-secretsmanager" }
```

Comment:
These packages have now been officially released on crates.io: e.g. https://crates.io/crates/aws-config
so better to use the latest stable release instead of these old alpha pre-releases from github